src/plugins/text/render.js

```diff
@@ -26,7 +26,7 @@ export function renderNode({ node, children, attributes }, editor, next) {
     return next()
 }
 
-export function renderMark({ mark, children, attributes }, editor, next) {
+export function renderDecoration({ mark, children, attributes }, editor, next) {
```

Comment:
It's rendering `span` instead correct mark tag

Reply:
You can still use `renderMark` here. From what I understand it's no longer used for rendering decoration, but it's still correct function.